src/lib/utils/remark-plugins/remark-link-card.ts

```diff
@@ -2,7 +2,7 @@ import type { Parent, Node } from "unist";
 import { visit } from "unist-util-visit";
 import { encode } from "html-entities";
 import ogs from "open-graph-scraper";
-import type { ImageObject } from "open-graph-scraper/dist/lib/types";
+import type { ImageObject } from "open-graph-scraper/types";
```

Comment:
ref: https://github.com/jshemas/openGraphScraper?tab=readme-ov-file#types-and-import-example